flexiblepricing/views/v0/__init__.py

```diff
@@ -41,6 +41,19 @@ def get_queryset(self):
         return models.FlexiblePrice.objects.filter(user=self.request.user).all()
 
 
+class FlexiblePriceCoursewareViewSet(ModelViewSet):
+    serializer_class = serializers.FlexiblePriceCoursewareAdminSerializer
+    authentication_classes = (SessionAuthentication, TokenAuthentication)
+    permission_classes = (IsAdminUser,)
+
+    def get_queryset(self):
+        return (
+            models.FlexiblePrice.objects.filter(user=self.request.user)
```

Comment:
This filtering limits the list to just the currently logged in user's requests. For the staff dashboard, this should be everyone's requests. (On my local instance, I had two requests: one for my admin account in a program and one for a regular learner in a different program. I can only see the program associated with my admin account's request in the dropdown.)